### PR TITLE
Add '-ascending' argument to gdb

### PR DESCRIPTION
**Pull request by marusak**
Use argument -ascending in gdbs "thread apply all" so the crashing thread is
depicted first.

Closes abrt/retrace-server#77.

Signed-off-by: Matej Marusak <mmarusak@redhat.com>

### CODE REVIEW

File: src/plugins/fedora.py [hunk 1]
```diff
@@ -4,6 +4,9 @@
 abrtparser = re.compile("^Fedora release ([0-9]+) \(([^\)]+)\)$")
 guessparser = re.compile("\.fc([0-9]+)")
 displayrelease = "Fedora release"
+gdb = "gdb"
```
Comment (jfilak) [inline src/plugins/fedora.py]: I would call this member 'gdb_package' to avoid some confusion.

File: src/plugins/fedora.py [hunk 2]
```diff
@@ -4,6 +4,9 @@
...
+gdb_run = "gdb -batch "
+gdb_stop = ""
```
Comment (jfilak) [inline src/plugins/fedora.py]: I would replace these two members with `gdb_executable` holding either `/usr/bin/gdb` or `/opt/rh/devtoolset-4/root/usr/bin/gdb`

Comment (marusak) [inline src/plugins/fedora.py]: This is great! Will change.

**Comment by jfilak** [comment]
Thank you!

**Pull request merged**